packages/socket-mode/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@slack/socket-mode",
-  "version": "2.0.0",
+  "version": "2.0.1",
```

Comment:
You can do this later (when you cut a new patch version)

Reply:
Good call and will do! o7